src/main/java/com/google/api/generator/gapic/composer/rest/HttpJsonServiceStubClassComposer.java

```diff
@@ -380,9 +384,74 @@ protected Expr createTransportSettingsInitExpr(
     return Collections.singletonList(expr);
   }
 
+  // Generates expr.methodName();
+  private MethodInvocationExpr invokeMethodfromVar(Expr expr, String methodName) {
```

Comment:
YOu probably don't need this method and use the MethodInvocationExpr construction code directly where you need it It will make it a bit more verbose, but oddly enough it will make it more idiomatic for gapic-generator-java. The same applies to the staticBuilder() method below